packages/@lwc/integration-karma/test/component/LightningElement.style/index.spec.js

```diff
@@ -0,0 +1,14 @@
+import { createElement } from 'lwc';
+
+import Override from 'x/override';
+import Test from 'x/test';
+
+it('should return the CSSStyleDeclaration of Host Element', () => {
+    const elm = createElement('x-test', { is: Test });
+    expect(elm.test).toEqual('X-TEST');
+});
+
+it('should be overrideable', () => {
+    const elm = createElement('x-override', { is: Override });
+    expect(elm.tagName).toEqual('x-foo');
+});
```

Comment:
These tests make sense for `tagName`, but they don't verify the `style` at all. A better test would be to confirm that, for example `elm.style.color = 'red'` sets the `computedStyle` to be the color red.